File: BauFahrplanMonitor.Core/Dto/ZwlMassnahmeOverlayDto.cs
namespace BauFahrplanMonitor.Core.Dto;

public sealed class ZwlMassnahmeOverlayDto {
    public string VonRl100 { get; init; } = default!;
    public string BisRl100 { get; init; } = default!;

    public DateTime MassnahmeBeginn { get; init; }
    public DateTime MassnahmeEnde   { get; init; }

    public string Regelungen  { get; init; } = default!;
    public string VzgListe    { get; init; } = default!;
    public bool   Durchgehend { get; init; }
    public string Zeitraum    { get; init; } = default;
}

File: web/src/models/ZwlMassnahmeOverlayDto.ts
export type ZwlMassnahmeOverlayDto = {
    vonRl100: string;
    bisRl100: string;
    massnahmeBeginn: string; // ISO-String
    massnahmeEnde: string;   // ISO-String
    regelungen: string;
    vzgListe: string;
    durchgehend: boolean;
    zeitraum?: string;
};
